### PR TITLE
Updated examples and installation YAMLs for AMQ Streams 1.6.0

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        strimzi.io/kind: cluster-operator
+    spec:
+      serviceAccountName: strimzi-cluster-operator
+      volumes:
+        - name: co-config-volume
+          configMap:
+            name: strimzi-cluster-operator
+      containers:
+        - name: strimzi-cluster-operator
+          image: strimzi/operator:0.20.0
```
Comment (kyguy) [inline install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:26]: Let's update these image names as part of this PR as well! e.g. registry.redhat.io/amq7/amq-streams-rhel7-operator:1.6.0

**Review by kyguy — APPROVED** [review]
Looks good! We just need to update the image names in the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml` file

File: install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml (hunk 1 above)
Comment (ppatierno) [reply install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:26]: good spot!

File: install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+              value: strimzi/kafka-bridge:0.19.0
+            - name: STRIMZI_DEFAULT_JMXTRANS_IMAGE
+              value: strimzi/jmxtrans:0.20.0
```
Comment (scholzj) [inline install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:86]: You will change these later?

Comment (ppatierno) [reply install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml:86]: nope they have to be changed! damn! that was a new file :-)

**Review by scholzj — APPROVED** [review]

**Pull request merged**